src/constants/aws.ts

```diff
@@ -10,13 +10,13 @@ export const getSubmitPackingUrl = (
         url += `&config=${config}`;
     }
     return url;
-}
+};
 
 //job status
-export enum JobStatus {
-    SUBMITTED = "SUBMITTED",
-    STARTING = "STARTING",
-    RUNNING = "RUNNING",
-    DONE = "DONE",
-    FAILED = "FAILED",
-}
\ No newline at end of file
+export const JOB_STATUS = {
+    SUBMITTED: "SUBMITTED",
+    STARTING: "STARTING",
+    RUNNING: "RUNNING",
+    DONE: "DONE",
+    FAILED: "FAILED",
+};
```

Comment:
I changed the casing and the type from an enum to a constant, just to match our other constants. Open to suggestion if this looks better or not, I can change it back if we don't like it